ui/.eslintrc

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
documenting what's this for.

Reply:
Add a linting section in README.md under ui directory to cover this.